cargo-near/src/main.rs

```diff
@@ -40,7 +40,7 @@ fn main() -> CliResult {
     let config = near_cli_rs::config::Config::get_config_toml()?;
     let global_context = near_cli_rs::GlobalContext {
         config,
-        teach_me: false,
+        verbosity: near_cli_rs::Verbosity::Interactive,
```

Comment:
looks like it's missing 
```rust
    let verbosity = match cli.teach_me{

       true => Verbosity::TeachMe,
      false => Verbosity::Interactive,
    };
    ```

Reply:
To be honest, I feel like I don't need actually to change anything as it's parsed later: https://github.com/near/cargo-near/blob/5d8e4c4371340935562c52a6a21945e57d87ab57/cargo-near/src/main.rs#L57

Reply:
It also works 
![image](https://github.com/user-attachments/assets/ddd5248b-f324-4cc5-9dbe-aea848ef7f08)

Reply:
K, the main question was whether the flag was enabled for the deploy chain for near-cli-rs. I don't feel like digging it too much myself now, so i'll just merge it and then revisit if someone notices any kind of strange inconsistent behaviour